metagenomics.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+import argparse
+import gzip
+import shutil
+import util.file
+import tools.kraken
+import tools.krona
+import tools.diamond
+import tools.picard
+__commands__ = []
+
+
+def kraken(inBam, db, outReport=None, outReads=None,
+           filterThreshold=None, threads=1):
```

Comment:
Actually using it now. It's pretty important because kraken actually has good boost with threads.